dts/bindings/iio/adc/silabs,gecko-adc.yaml

```diff
@@ -0,0 +1,39 @@
+#
+# Copyright (c) 2019 Interay Solutions B.V.
+# Copyright (c) Oane Kingma
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+---
```

Comment:
You should remove these empty comment lines, as well as the one with `---`, for consistency with (almost all) other bindings.

Reply:
Done